core/controllers/base.py

```diff
@@ -215,6 +215,11 @@ def dispatch(self):
                 b'https://oppiatestserver.appspot.com', permanent=True)
             return
 
+        if (feconf.ENABLE_MAINTENANCE_MODE and
```

Comment:
Should we add unit test for these changes in BaseHandler test class? [Though there exist some integration test, but wanted to check whether we need to update the test as the behaviour and code-structure is a bit changed!]

Reply:
Yep, I will add them.